tests/test_rdf.py

```diff
@@ -18,24 +19,36 @@ def load_model(name):
         os.path.dirname(__file__), 'cellml_files', name))
 
 
+@pytest.fixture(scope="module")
+def test_simple_odes():
+    model = load_model('test_simple_odes')
+    return model
+
+
+@pytest.fixture(scope="module")
+def test_bad_annotations():
+    model = load_model('test_bad_annotations')
+    return model
+
+
 def test_create_rdf_node():
```

Comment:
Should really add some more low-level tests here showcasing the different possible forms for `node_content`. Maybe also check the return types are as expected?

Reply:
ok